### ISSUE TITLE
Set Recieved Amount Request

### ISSUE THREAD

**Issue by trymeouteh** [haven-protocol-org/haven-web-app#49]
Please add an option to request a certain amount on your receiving address and QR code for receiving funds.

**Comment by g45t345rt** [comment]
I think the QR code is already implemented.
Go to Transfer>Receive Tab and click the 'Show QR' button

![image](https://user-images.githubusercontent.com/49697006/121811184-9dedc000-cc31-11eb-811a-57248dfe7def.png)

**Comment by trymeouteh** [comment]
Yes but not able to set an amount to request and have the QR code embed the amount requested
